statsmodels/discrete/tests/test_discrete.py

```diff
@@ -1559,6 +1559,28 @@ def test_binary_pred_table_zeros():
     assert_equal(res.pred_table(), expected)
 
 
+def test_unchanging_degrees_of_freedom():
+    # see GH3734
+    data = sm.datasets.randhie.load()
+    model = sm.NegativeBinomial(data.endog, data.exog, loglike_method='nb2')
+    
+    res1 = model.fit()
```

Comment:
in general: If convergence is not of interest in a test, then use start_params with solution, or close to solution.
that saves some time in the test runs